src/controller/CHIPDeviceController.h

```diff
@@ -60,6 +60,7 @@ class DLL_EXPORT ChipDeviceController : public SecureSessionMgrCallback
     void * AppState;
 
     CHIP_ERROR Init(NodeId localDeviceId);
+    CHIP_ERROR Init(NodeId localDeviceId, System::Layer * systemLayer, InetLayer * inetLayer);
```

Comment:
some doc comments on when one Init should be used vs another would be helpful.

Is the first version useful if CONFIG_DEVICE_LAYER is not defined? How aboud ifdef-ing which version of init is available based compile flags?

Reply:
Added a ticket to track unifying the Init functions into a single meaningful one -- #1902.

Reply:
Added docs as well.